doc/source/serve/http-servehandle.rst

```diff
@@ -152,6 +152,43 @@ To try it out, save a code snippet in a local python file (i.e. main.py) and in
     ray start --head
     python main.py
 
+.. _serve-http-adapters:
+
+HTTP Adapters
+^^^^^^^^^^^^^
+
+Ray Serve provides a suite of adapter for input schema conversion. You can directly
+import them and put them into your FastAPI app. Alternatively, just use it with
+:ref:`Ray AI Runtime (AIR) model wrapper<air-serve-integration>` feature to one click deploy pre-trained models.
```

Comment:
reverse the ordering of this: common path will be model wrappers, so lead with that

Reply:
same with the example please. first show how to use model wrapper, then fastapi API